benchmarks/benchmarks/bench_ufunc.py

```diff
@@ -36,13 +36,17 @@ def time_broadcast(self):
 
 class At(Benchmark):
     def setup(self):
-        self.vals = np.random.rand(1_000_000)
-        self.idx = np.random.randint(1000, size=1_000_000).astype(np.intp)
+        rng = np.random.default_rng()
```

Comment:
Since the previous version was pinned by `numpy.random.seed(1)` in `common.py`, would it make sense to pick some seed here?

Reply:
yes.
```suggestion
        rng = np.random.default_rng(1)
```